collections/vegetation-indices.yaml

```diff
@@ -127,7 +127,7 @@ Extent:
     interval:
       -
         - '2016-10-01T00:00:00Z'
-        - null
+        - '2021-02-28T00:00:00Z'
```

Comment:
Adjust the hr,min,sec part  , `'2021-02-28T00:00:00Z' ` leaves out some data

 something like  '2021-02-28T23:60:60Z'  would capture all data if i am not wrong

Reply:
Changed to `2021-03-01T00:00:00Z` as I feel it's more correct.

Reply:
alright .. looks good

lets merge